### ISSUE TITLE
Food for thought: Removing AntD

### ISSUE THREAD

**Issue by ahmadalfy** [egytech-fyi/egytech-fyi#26]
[Ant Design](https://ant.design/) is a great UI library, but I believe it has a huge impact on the application. According to [vite-bundle-analyzer](https://www.npmjs.com/package/vite-bundle-visualizer), it contributes to nearly 25% of the application bundle size:

![_private_var_folders_l8_zhhz62y15dd08z9x2pl18c780000gn_T_tmp-71172-nYIq7enE1gb3_stats html](https://github.com/egytech-fyi/egytech-fyi/assets/87611/abc0a993-df80-41a4-b3b9-c3f734ae35bc)

<img width="437" alt="image" src="https://github.com/egytech-fyi/egytech-fyi/assets/87611/eab341e3-a8c4-476a-a2eb-c6f0b1496920">

That's 500kb of JavaScript the users have to download and the browser have to parse and execute. The used components are:

- Layout container
- Button
- Drawer
- Typography
- Menu
- Form Elements (inputs, select, checkboxes ... etc)
- Divider

I feel it's a bit of an overkill and they should be replaced with native controls and elements. It requires a little bit of work but it will be impactful.

**Comment by MrViometal** [comment]
@ahmadalfy I agree, I have been thinking about making our own UI library to make our lives easier on the long run, but I didn't have time when first starting the project of course, and also cause I believed we will have to have a redesign eventually when some UI designers volunteer to build us a design system.

Maybe we can skip the wait for the UI designers and just build a native UI library according to one of the already available design systems out there 👍🏼

**Comment by abdallahalaa21** [comment]
@MrViometal  you can use something like [shadcn](https://ui.shadcn.com/ ) , and I can work with you on this ui library 
we can also use tailwind CSS and shadcn is also compatible with tailwind CSS

**Comment by OSA98** [comment]
Hey @MrViometal 

Maybe we can further discuss how we can collaborate on that. I would be happy to contribute.

Here is my behance : https://www.behance.net/osamaabdelnaser1

Check your twitter dms

**Comment by MarkMagdyAziz** [comment]
@MrViometal @mahsayedsalem Hello guys, I can contribute and support here

**Comment by MrViometal** [comment]
@MarkMagdyAziz hey mark, would love to hear about how can you help in that regard, could you tell me if it is design or development? Also if you can elaborate how it would be much appreciated 🙏🏻

**Comment by MarkMagdyAziz** [comment]
@MrViometal I can contribute the development migration to chadcn or native UI

**Comment by MrViometal** [comment]
@MarkMagdyAziz okay, I would suggest that you open a PR with minimal changes as a POC in order judge the process before committing to it, and if we agree then we can start working on the whole migration

**Comment by MarkMagdyAziz** [comment]
comments have been resolved Thank youu

**Comment by MarkMagdyAziz** [comment]
@MrViometal I have already opened a PR as POC kindly check
